grub-core/kern/dl.c

```diff
@@ -680,6 +688,19 @@ grub_dl_load_file (const char *filename)
   void *core = 0;
   grub_dl_t mod = 0;
 
+#ifdef GRUB_MACHINE_EFI
+  if (grub_efi_secure_boot ())
+    {
+#if 0
```

Comment:
I know this is from an existing patch but since we don't use mkconfig perhaps we can turn it on or make it optional.